styles/base/_reset.scss

```diff
@@ -1,58 +1,6 @@
+@use 'normalize.css/normalize';
 @use '../mixins/reset';
 
-/*
-  Consistency fixes
-  adopted from http://necolas.github.com/normalize.css/
```

Comment:
I note this URL was obsolete (the current one is https://necolas.github.io/normalize.css/).

Reply:
Yeah, I found that out the hard way, too